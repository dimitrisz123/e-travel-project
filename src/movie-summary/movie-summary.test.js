import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import MovieSummary from "./movie-summary";

it("renders MovieSummary correctly", () => {
	const details = [
		{
			fields: {
				title: "a new hope",
				opening_crawl: "a great movie",
				director: "dim zer",
				episode_id: "6"
			}
		}
	];
	const wrapper = shallow(<MovieSummary details={details} />);
	expect(toJson(wrapper)).toMatchSnapshot();
});
