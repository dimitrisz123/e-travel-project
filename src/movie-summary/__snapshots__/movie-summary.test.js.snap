// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MovieSummary correctly 1`] = `
<div>
  <div
    className="summary-wrapper"
    key="6"
  >
    <p
      className="summary-title"
    >
      a new hope
    </p>
    <p
      className="summary-opening_crawl"
    >
      a great movie
    </p>
    <p
      className="summary-director"
    >
      Directed by: 
      dim zer
    </p>
  </div>
</div>
`;
